entity-api-spec.yaml

```diff
@@ -1,396 +1,1267 @@
           required: true
           schema:
             type: string
       responses:
-        "200":
-          description: successful operation
-        "400":
+        '200':
+          description: The samples were successfully created.  An array of objects each containing a uuid, hubmap_id and submission_id associated with each newly created sample.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    hubmap_id:
+                      type: string
+                    submission_id:
+                      type: string
+                    uuid:
+                      type: string
+        '400':
           description: Invalid json input
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "500":
+        '401':
+          description: The user's token has expired or the user did not supply a valid token
+        '500':
           description: Internal error
-
-  /entities/multiple-samples:
+  '/collections/{collection_uuid}}/add-datasets':
     put:
-      summary: 'Update properties of multiple samples via one API request'
-      responses:
-        "200":
-          description: successful operation
-        "400":
-          description: Invalid json input
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "404":
-          description: The target entity could not be found
-        "500":
-          description: Internal error
-
-  /collections/{collection_uuid}}/add-datasets:
-    put:
-      summary: 'Link the given list of datasets to the target collection'
+      summary: Link the given list of datasets to the target collection
       parameters:
         - name: collection_uuid
           in: path
-          description: 'The UUID of target collection'
+          description: The UUID of target collection
           required: true
           schema:
             type: string
       responses:
-        "200":
+        '200':
           description: successful operation
-        "400":
+        '400':
           description: Invalid or misformatted collection uuid
-        "401":
-          description: "The user's token has expired or the user did not supply a valid token"
-        "404":
+        '401':
+          description: The user's token has expired or the user did not supply a valid token
+        '403':
+          description: The user is not authorized to add datasets to the collection
+        '404':
           description: The target entity could not be found
-        "500":
+        '500':
           description: Internal error
-          
-  /collection/redirect/{id}:
-    put:
-      summary: 'Redirect a request from a doi service for a collection of data'
+  '/doi/redirect/{id}':
```

Comment:
The current entity-api also supports the old endpoint `/collection/redirect/<id>`

https://github.com/hubmapconsortium/entity-api/blob/test-release/src/app.py#L1644

Should we add it to the API docs too?

Reply:
Same thing with this one, I purposely left it out of the documentation because it is "deprecated forever".